charts/stable/whoogle/values.yaml

```diff
@@ -0,0 +1,35 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  repository: benbusby/whoogle-search
+  pullPolicy: IfNotPresent
+  # tag defaults to appVersion in Chart.yaml
+  # tag:
+
+strategy:
+  type: Recreate
+
+# See more environment variables in the whoogle documentation
+# https://github.com/benbusby/whoogle-search#environment-variables
+env:
+  # WHOOGLE_CONFIG_COUNTRY:
+  WHOOGLE_CONFIG_DARK: 1
+  # WHOOGLE_CONFIG_SAFE:
+  # WHOOGLE_CONFIG_ALTS:
+  # WHOOGLE_CONFIG_TOR:
+  # WHOOGLE_CONFIG_NEW_TAB:
+  # WHOOGLE_CONFIG_GET_ONLY:
+  WHOOGLE_CONFIG_URL: https://<your url>/
+  # WHOOGLE_CONFIG_STYLE:
```

Comment:
What's the reasoning for defaulting some values, but commenting out other ENV vars? I'd leave all of them commented, and leave it up to the user to set their preferences.